custom_components/tuya_local/devices/ultenic_air_fryer.yaml

```diff
@@ -86,25 +85,25 @@ secondary_entities:
         name: sensor
         mapping:
           - dps_val: "0"
-            value: Standby
+            value: standby
```

Comment:
These were capitalised as there is no translation for sensor values, so they will appear in the UI as-is.